.vscode/settings.json

```diff
@@ -14,5 +14,6 @@
     "**/.factorypath": true,
     "**/*~": true
   },
-  "C_Cpp.default.configurationProvider": "vscode-wpilib"
+  "C_Cpp.default.configurationProvider": "vscode-wpilib",
+  "C_Cpp.errorSquiggles": "disabled"
```

Comment:
delete this line. Error squiggles need to be turned on. You can't just ignore your errors.